ui/perfherder/alerts/AlertHeader.jsx

```diff
@@ -66,8 +85,9 @@ const AlertHeader = ({
     );
     return issueTrackerUrl + alertSummary.bug_number;
   };
-  const handleRevertRevision = async () => {
-    await changeRevision(alertSummary.original_revision);
+  const handleRevertRevision = (pushORfrom) => () => {
```

Comment:
What do you think of using `revisionType` instead of `pushORfrom`

Reply:
Agree